nikola/__main__.py

```diff
@@ -363,7 +371,33 @@ def run(self, cmd_args):
                 LOGGER.error("This command needs to run inside an "
                              "existing Nikola site.")
                 return 3
-        return super(DoitNikola, self).run(cmd_args)
+        try:
```

Comment:
Looking at it better, I would have to exit(1) in the next layer, since this run() prints the exceptions.